pinot-controller/src/main/java/com/linkedin/pinot/controller/api/resources/PinotSegmentUploadRestletResource.java

```diff
@@ -321,7 +321,7 @@ private SuccessResponse uploadSegment(FormDataMultiPart multiPart, boolean enabl
           .validateSegment(segmentMetadata, tempSegmentDir);
 
       // Zk operations
-      completeZkOperations(enableParallelPushProtection, headers, tempDecryptedFile, provider, segmentMetadata,
+      completeZkOperations(enableParallelPushProtection, headers, tempEncryptedFile, provider, segmentMetadata,
```

Comment:
This is a bug fix right? Can you please add it into the commie message?

Reply:
Yes, this is a bug fix, done.